bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_ttree.py

```diff
@@ -80,4 +201,8 @@ def pythonize_ttree(klass, name):
         klass._OriginalBranch = klass.Branch
         klass.Branch = _Branch
 
+    if name == 'TTree':
```

Comment:
You can move this up with the rest of the pythonisations, no need for an extra check only for `TTree`, even if it will be injected for `TChain` as well. Anyway, `TChain` will inherit the pythonisation if you just inject it for `TTree`.

Reply:
Good point! I need to check whether this actually works for a `TChain`...

Reply:
I'll add a test.

Reply:
Fixed!